ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -755,106 +796,3 @@ objects:
-      - "console-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1"
-      - "downloads-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1"
-      - "grafana-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1"
-      - "oauth-openshift.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1"
-      - "prometheus-k8s-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1"
-    - ip: "${BASTION_SVC_1_2_IP}"
-      hostnames:
-      - "alertmanager-main-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "canary-openshift-ingress-canary.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "console-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "downloads-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "grafana-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "oauth-openshift.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-      - "prometheus-k8s-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2"
-    - ip: "${BASTION_SVC_1_3_IP}"
-      hostnames:
-      - "alertmanager-main-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "canary-openshift-ingress-canary.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "console-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "downloads-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "grafana-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "oauth-openshift.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-      - "prometheus-k8s-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3"
-    - ip: "${BASTION_SVC_1_4_IP}"
-      hostnames:
-      - "alertmanager-main-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "canary-openshift-ingress-canary.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "console-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "downloads-openshift-console.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "grafana-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "oauth-openshift.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
-      - "prometheus-k8s-openshift-monitoring.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4"
```

Comment:
I don't think we can remove these - I believe they handle the resolution from the prow cluster to the remote openshift cluster - dnsmasqOptions won't change that, since that handles internal to openshift network resolution.